runtime/kusama/src/lib.rs

```diff
@@ -1541,6 +1637,56 @@ impl OnRuntimeUpgrade for CrowdloanIndexMigration {
 	}
 }
 
+/// Initialize with-Polkadot bridge.
+pub struct InitializePolkadotBridge;
```

Comment:
I'm unsure about this upgrade. We'll need to initialize the "pallet owner" field of both bridge pallets (that's the account that may issue some special "requests" - e.g. halt bridge, update conversion rate, ...). Don't know if doing this during runtime upgrade or using some governance mechanisms after runtime upgrade is better here.